### PR TITLE
[TASK] Improve user TSconfig options

### PR DESCRIPTION
**Pull request by linawolf**
- correct indentation
- introduce names confvals

releases: main, 12.4, 11.5

### CODE REVIEW

File: Documentation/UserTsconfig/Options.rst [hunk 1]
```diff
@@ -49,8 +50,8 @@ alertPopups
 -----------
 
 ..  confval:: alertPopups
-
-    :Data type: bitmask
+    :name: useroptions-alertPopups
+    :type: bitmask
```
Comment (brotkrueml) [inline Documentation/UserTsconfig/Options.rst:54]: Why changing to "type"? We use "Data type" in TypoScript and TSconfig throughout the manual?

Comment (linawolf) [reply Documentation/UserTsconfig/Options.rst:54]: type is a native property of the confval and always displayed on top. We can change its label globally if desired.

`:Data type:` is just an arbitary label without meaning behind it. Whereever I have seen it I have changed it or introduced the new one right away. And we can make a mass-change to catch those not caught in the end. Same goes with switching from definition list (with empty line after confval) to real parameters.

Comment (brotkrueml) [reply Documentation/UserTsconfig/Options.rst:54]: Okay, so you take care about changing all the "Data type" to "type" in the next days? As this wasn't announced in the PR description, I commented about it.

Comment (linawolf) [reply Documentation/UserTsconfig/Options.rst:54]: I have been unifying the confvals all over these  changes. This is one of the many inconsitencies I found. All my PRs had these kind of changes in them. It is an iterative process of getting them all on the same level. But as you can see I am working on it.

The way that Parameters where until now displayed in the TSconfig was inconsistent all over

Comment (linawolf) [reply Documentation/UserTsconfig/Options.rst:54]: I just checked, there are currently 102 usages of `:type:` and only 44 usages of `:Data type:` in this reference. All usages of the latter are on the page this PR adresses

Comment (brotkrueml) [reply Documentation/UserTsconfig/Options.rst:54]: Okay, thanks, TypoScript has far more "Data type" usages.

**Pull request merged**